scalableminds/webknossos
# adds buttons for switching organizations for workflows

### Steps to test:
- Make a second organization
- Run a vx workflow
- Open workflow report in wk
- Switch to other organization
- See the "switch to org" screen and click the button
- Voilà

------
